specification/resources/resource-manager/Microsoft.Resources/stable/2017-05-10/resources.json

```diff
@@ -34,6 +34,321 @@
     }
   },
   "paths": {
+    "/subscriptions/{subscriptionId}/providers/Microsoft.Resources/deployments/{deploymentName}": {
+      "delete": {
+        "tags": [
+          "Deployments"
```

Comment:
>Deployments [](start = 11, length = 11)

Deployments_SubscriptionLevel ? (or something better, to differentiate from the original APIs)